content/tutorial/tutorial.md

```diff
@@ -12,125 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Deze tutorial vereist geen voorbestaande React kennis.
```

Comment:
Should we translate the word `tutorial` as well? An option for this would be `inleiding` (introduction).

```suggestion
Deze inleiding vereist geen voorbestaande React kennis.
```

Do we have any guidelines for this? Can we/should we start a glossary?

Reply:
We have a (partial) glossary #5

Reply:
> Should we translate the word `tutorial` as well? An option for this would be `inleiding` (introduction).

`tutorial` is a [valid Dutch (loan)word](https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/tutorial). However if we insist on translating it, maybe `les` (lesson) would be a better translation than `inleiding`?

Reply:
Since `tutorial` isn't yet included in the glossary #5 , I'd be in favor of adding it to the keep as-is list. I think part of the meaning is lost in translation with `inleiding`, `les`, etc.

Reply:
assume => gaat niet uit van
voorbestaande kennis => voorkennis

```suggestion
Deze tutorial gaat niet uit van enige voorkennis van React.
```